src/main/java/com/minecolonies/structures/lib/RenderUtil.java

```diff
@@ -0,0 +1,56 @@
+package com.minecolonies.structures.lib;
+
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.util.Mirror;
+import net.minecraft.util.Rotation;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+public final class RenderUtil
+{
+
+    private RenderUtil()
+    {
+        throw new IllegalArgumentException("Utility Class");
+    }
+
+    public static void applyRotationToYAxis(@NotNull final Rotation rotation)
+    {
+        float angle = 0f;
+        switch (rotation)
+        {
+            case NONE:
+                angle = -0f;
+                break;
+            case CLOCKWISE_90:
+                angle = -90f;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 90f to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Upper-case this literal "f" suffix. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS818)